tests/src/Interop/IJW/NativeCallingManaged/IjwNativeCallingManagedDll/IjwNativeCallingManagedDll.cpp

```diff
@@ -0,0 +1,21 @@
+#pragma managed
```

Comment:
License header

Reply:
Fixed here and in a couple other files I missed in the last change.